### PR TITLE
Added more logging + increased delay after creating a fresh branch

### LINKED ISSUE #166: Bug: sometimes gitlab api fails with `ValidationError: "id" is required`
Example: https://github.com/paritytech/cumulus/pull/2220#issuecomment-1440899197
```
@mordamax Command "$PIPELINE_SCRIPTS_DIR/commands/bench/bench.sh" all has finished. Result:

ValidationError: "id" is required
ValidationError: "id" is required
{"message":{"base":["Reference not found"]}}
```

### PR DESCRIPTION
**Pull request by mordamax**
Aims to close #166

### CODE REVIEW

File: src/gitlab.ts [hunk 1]
```diff
@@ -118,9 +118,13 @@ export const runCommandInGitlabPipeline = async (ctx: Context, task: Task): Prom
   */
   let wasBranchRegistered = false;
   const waitForBranchMaxTries = 3;
-  const waitForBranchRetryDelay = 1024;
+  const waitForBranchRetryDelayMs = 2000;
```
Comment (Bullrich) [inline src/gitlab.ts:121]: I'm mentally bothered that it is `2000` and no `2048` haha

But yeah, it's two seconds so it's a rounder number

**Review by Bullrich — APPROVED** [review]
Seems sensible!

**Review by mutantcornholio — CHANGES_REQUESTED** [review]
This PR does throwing errors and logging errors at the same time, which results in double logging, or logging successfully handled errors. 
Can we change it the way that we'll log the errors when handling those?

**Comment by mordamax** [comment]
> This PR does throwing errors and logging errors at the same time, which results in double logging, or logging successfully handled errors. Can we change it the way that we'll log the errors when handling those?

good point, rolled back and added error log on the common handler. 
So before in any case (error or ok) it was logging just debug, and it wasn't clear if there's an error happened. Now this will be noticeable

Thanks

**Review by mutantcornholio — APPROVED** [review]

**Pull request merged**